syntax = "proto3";

message Mob {
  int32 id = 1;
  int32 health = 2;
  int32 position_x = 3;
  int32 position_y = 4;
  bool in_battle = 5;
  int32 parent_id = 6;
}

message MobList {
  repeated Mob mobs = 1;
}
